newIDE/app/src/ProjectCreation/services/BrowserCreation.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import { t } from '@lingui/macro';
+import { type I18n as I18nType } from '@lingui/core';
+
+import UrlStorageProvider from '../../ProjectsStorage/UrlStorageProvider';
+import { type StorageProvider, type FileMetadata } from '../../ProjectsStorage';
+import { getExample } from '../../Utils/GDevelopServices/Example';
+import { sendNewGameCreated } from '../../Utils/Analytics/EventSender';
+import { showErrorBox } from '../../UI/Messages/MessageBox';
+import { type ExampleShortHeader } from '../../Utils/GDevelopServices/Example';
+const gd: libGDevelop = global.gd;
+
+// Signatures of this function and its arguments contain useless arguments
+// because this function has to match the signature of LocalCreation.onCreateBlank
+// that needs these arguments.
+export const onCreateBlank = (
+  onOpenCallback: (
+    project: gdProject,
+    storageProvider: ?StorageProvider,
+    fileMetadata: ?FileMetadata
+  ) => void
+) => async (i18n: I18nType, outputPath?: string) => {
+  sendNewGameCreated('');
+
+  const project = gd.ProjectHelper.createNewGDJSProject();
+  onOpenCallback(project, null, null);
+};
+
+export const onCreateFromExampleShortHeader = (
+  isOpeningCallback: boolean => void,
+  onOpenCallback: (
+    storageProvider: StorageProvider,
+    fileMetadata: FileMetadata
+  ) => void
+) => async (i18n: I18nType, exampleShortHeader: ExampleShortHeader) => {
```

Comment:
See my other comment, could we:
- go from a callback to a promise with a returned object?
- collapse the high order function into a single function with an object as arg: `{ setIsOpening, i18n, exampleShortHeader }`

Could simplify how we use these functions :)